### ISSUE TITLE
Using Absolute Image Name in COPY --from

### ISSUE THREAD

**Issue by rawkode** [GoogleContainerTools/kaniko#297]
`COPY --from=` only supports stages represented within the Dockerfile. It should probably support arbitrary images too:

```
COPY --from=composer:1.6 /usr/bin/composer /usr/bin/composer
```

Happy to take a shot at this one. I'll see if I can get this to work

**Comment by rawkode** [comment]
Also, probably not that urgent. One can add a faux-layer:

```
FROM composer:1.6 AS composer

FROM another_base

COPY --from=composer
```

**Comment by dlorenc** [comment]
This was fixed recently in #479
